gef.py

```diff
@@ -303,9 +304,25 @@ def gef_print(x="", *args, **kwargs):
     return print(x, *args, **kwargs)
 
 
+@contextlib.contextmanager
+def dont_buffer():
+    global __gef_int_stream_buffer__
+    old_buffer = __gef_int_stream_buffer__
+    __gef_int_stream_buffer__ = None
+    try:
+        yield
+    finally:
+        __gef_int_stream_buffer__ = old_buffer
+
+def get_cmd_output(cmd):
+    with dont_buffer():
+        return gdb.execute(cmd, to_string=True)
+
+
 def bufferize(f):
     """Store the content to be printed for a function in memory, and flush it on function exit."""
 
+    return f # DELETEME
```

Comment:
?

Reply:
This is becausef of #362. See my TODOs at the top.